src/lib.js

```diff
@@ -257,7 +261,7 @@ export default class Migrator {
           name: migrationName,
           createdAt: timestamp
         });
-        return createdMigration.toJSON();
+        return createdMigration;
```

Comment:
Can you tell me about the intent of the toJSON() here? How could this ever have worked since a promise does not have a toJSON() on it? Did you mean to call `.then(m => m.toJSON())` ? I'm happy to put it back in as long as it's not blowing up every time a new migration is syned to the DB. Who is using the JSON that is supposed to be returned, and does that caller expect a Promise that resolves to a String (JSON) or does it expect a literal String (JSON)?